src/app/app.ts

```diff
@@ -150,6 +150,51 @@ export function view(state: IState, actions: any) {
       id: 'app',
       class: classNames({'theme-dark': state.settings.darkTheme}),
       oncreate: (el: El) => actions.onCreate(el),
+      ondragover: (ev: Event) => {
```

Comment:
Can you move this logic for `ondragover`, `ondragleave`, `ondrop` into the actions object so the view function isn't so long.

Reply:
See line 152

Reply:
@jiahongdong Can you look at the first comment in this section. It would best if we moved this logic to be beside the other logic in the actions section at the top of this file